README.md

```diff
@@ -405,3 +406,10 @@ Include the next release leads in the PR as a reminder.
 
 Send a PR like [this one](https://github.com/knative-sandbox/knobots/pull/18) to
 bump knobots auto release workflow to the next release.
+
+## Release docs
```

Comment:
Are the release leads supposed to perform the documentation release after all other repos have been released ?

How it is ensured that the release leads have the proper permissions for doing this ? (i.e. who is contacting @carieshmarie or @RichieEscarez for setting the SSH keys for Netlify and GitHub ?)

Reply:
Hi Roland, 

A1: Yes, wait for /Serving and /Eventing (see steps below).

A2: You dont necessarily need elevated permissions or special SSH keys but having Approval privileges for the /Docs and /Website repos helps ease the process (just requires the "releaser" to get an LGTM).  

Today, Netlify and GitHub just work. You dont need to touch Netlify, the knative.dev builds automatically run, pickup, and show whatever is merged in GitHub (from both /docs and /website repos). I was able to hand off the process to both @csantanapr and @omerbensaadon (who "released" the last two versions).

## General steps

As a minimum example, here are the two PRs from the last release, that you can use to "cut and publish a new release":

1. Wait for dependent repos to release (today thats /serving and /eventing), then generate the new API docs and merge them before creating a new branch: https://github.com/knative/docs/pull/3426
2. Create the new /docs branch: https://knative.dev/help/maintainer/docs-release-process/#create-branch-from-updated-main
3. Add the new /docs branch to the website builds: https://github.com/knative/website/pull/287

With that said, the user experience team is completely changing the Docs build and publishing process but they are defining/driving that effort and can help you understand whats changed (ie the new steps to cut a release).